### DISCUSSION TITLE
GLB Optimizer and Texture Converter (WEBP, KTX2)

### DISCUSSION THREAD

**Discussion by eldinor** [donmccurdy/glTF-Transform#1453] [Show and tell]
Hello friends,

**Here is GLB Optimizer - https://glb.babylonpress.org/**

Made with GLTF-Transform and Babylon.js Sandbox.
Supports ETC1S and UASTC texture conversion.
Supports pixel comparison.
![image](https://github.com/donmccurdy/glTF-Transform/assets/25833327/dc52892d-aff5-4a4c-91b1-cdd120e6a413)

More info and how to use - https://forum.babylonjs.com/t/glb-optimizer-for-geometry-and-texture-conversion-webp-and-ktx2/47416

**Comment by riccidiccivj-jpg** [comment]
Hi did the domain stop working? it's unreachable

**Comment by eldinor** [comment]
Seems it is working without issues  (at least I don't see any).
Check also https://batch.babylonpress.org/ for batch GLB optimizing.
